concent_api/middleman/middleman_server.py

```diff
@@ -57,7 +61,8 @@ def _run(self) -> None:
             exit(ERROR_ADDRESS_ALREADY_IN_USE)
         try:
             # Serve requests until Ctrl+C is pressed
-            logger.info('MiddleMan is serving on {}'.format(self._server_for_concent.sockets[0].getsockname()))
+            logger.info('MiddleMan is serving for Concent on {}'.format(self._server_for_concent.sockets[0].getsockname()))
```

Comment:
you could break this line and the one below

Reply:
Done